src/api/link_post.js

```diff
@@ -7,6 +24,31 @@ const isValidDID = did => {
   return isIPFS.cid(parts[2])
 }
 
+const isValidAccountType = type => {
+  return Object.values(ACCOUNT_TYPES).includes(type)
+}
+
+const isSupportedChainId = chainId => {
+  return chainId && !!SUPPORTED_CHAINS[chainId]
```

Comment:
I Prefer to do `Boolean(var)` rather than `!!var` for clarity.

Reply:
Your code - your rules :D